express/parsers/apps/nwchem/formats/txt.py

```diff
@@ -0,0 +1,51 @@
+import os
+import io
+import re
+import numpy as np
+
+from express.parsers.utils import find_file
+from express.parsers.settings import Constant
+from express.parsers.apps.nwchem import settings
+from express.parsers.formats.txt import BaseTXTParser
+
+
+class NwchemTXTParser(BaseTXTParser):
+    """
+    Nwchem text parser class.
+    """
+
+    def __init__(self, work_dir):
+        super(NwchemTXTParser, self).__init__(work_dir)
+
+    def total_energy(self, text):
+        """
+        Extracts total energy.
+
+        Args:
+            text (str): text to extract data from.
+
+        Returns:
+             float
+        """
+        return self._general_output_parser(text, **settings.REGEX["total_energy"])
+
+    # NEED TO CHECK MATH/UNITS ON TOTAL ENERGY CONTRIBUTION
+    def total_energy_contributions(self, text):
+        """
+        Extracts total energy contributions.
+
+        Args:
+            text (str): text to extract data from.
+
+        Returns:
+            dict
+        """
+        energy_contributions = {}
+        for contribution in settings.TOTAL_ENERGY_CONTRIBUTIONS:
+            value = self._general_output_parser(text, **settings.TOTAL_ENERGY_CONTRIBUTIONS[contribution])
+            if value is not None:
+                energy_contributions.update({contribution: {
+                    'name': contribution,
+                    'value': value * 27.2114
```

Comment:
Don't we have a setting file with constants for `Ry_2_ev` and other things like that included there?

Reply:
I think there is I will check it for hartrees to ev, and update it accordingly along with these lines.